pylab/accounts/models.py

```diff
@@ -0,0 +1,17 @@
+import django.contrib.auth.models as auth_models
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(auth_models.User, related_name='profile')
+    language = models.CharField(_('Language'), max_length=7, choices=settings.LANGUAGES, default='lt', blank=True)
```

Comment:
I think, default language should be empty string, meaning, language will be detected from web browser headers instead of being hardcoded to `'lt'`.